extensions/amp-analytics/amp-analytics.md

```diff
@@ -16,79 +16,148 @@ limitations under the License.
+The `<amp-analytics>` element is used to measure activity on an AMP document. The details concerning what is measured and how
+that data is sent to an analytics server is specified in a JSON configuration object.
 
-This tag provides built-in support for most widely used analytics services. In addition, the tag allows for the analytics config to be specified such that the analytics data can be sent to any specified URL is a custom format.
+For example, the following `<amp-analytics>` element is configured to send a request to `https://example.com/analytics`
+when the document is first loaded, and each time an `<a>` tag is clicked:
 
-The tag is driven by JSON config that can be specified in one of the three places:
-
-**Remote configs**: This config is loaded in the page via a URL. The URL can be specified in the `config` attribute of the `amp-analytics` tag.
-**Inline configs**: This config is specified directly on the AMP page inside the `amp-analytics` tag.
-**Built in configs**: The AMP platform has these configs built into the platform. These can be loaded onto the page via the `type` attribute on the `amp-analytics` tag.
-
-If more than one methods of specifying the config are used, the configs are merged together in the order above (highest to lowest precedence).
-
-
-##### <a name="inline"></a>Inline Config with a custom host & url format
-```javascript
+```html
 <amp-analytics>
 <script type="application/json">
 {
-  "host": "my-analytics.com:8080",
   "requests": {
-    "base_hit": "/collect?v=1&_v=a0&aip=true&_s=HIT_COUNT&dl=DOMAIN&dt=TITLE&sr=SCREEN_WIDTHxSCREEN_HEIGHT&ht=TIMESTAMP&cid=CLIENT_IDENTIFIER&tid=ACCOUNT"
-    "pageview": "/r/{base_hit}&t=pageview&_r=1"
-  }
+    "pageview": "https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}",
+    "event": "https://example.com/analytics?eid=${eventId}&elab=${eventLabel}&acct=${account}"
+  },
   "vars": {
-    "account_id": "123456",
+    "account": "ABC123"
+  },
+  "triggers": {
+    "track pageview": {
+      "on": "visible",
+      "request": "pageview"
+    },
+    "track anchor clicks": {
+      "on": "click",
+      "selector": "a",
+      "request": "event",
+      "vars": {
+        "eventId": "42",
+        "eventLabel": "clicked on a link"
+      }
+    }
   }
-  "triggers": [{
-    "on": "LOAD",
-    "request": "pageview"
-  }]
 }
 </script>
 </amp-analytics>
 ```
-##### <a name="builtin"></a>Built-in tag for a service `XYZ` with inline config
+
+##### Configuration
+
+Configuration may be specified inline (as shown in the example above) or fetched remotely by specifying a URL in the
+`config` attribute. Additionally, built-in configuration for popular analytics vendors can be selected using
+the `type` attribute.
+
+If configuration data from more than one of these sources is used, the configuration objects will
+be merged together such that remote configuration takes precedence over inline configuration, and inline configuration
```

Comment:
@btownsend I thought the plan was for inline to have higher precedence than remote?